control-plane-operator/controllers/hostedcontrolplane/olm/assets/catalog-certified.deployment.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: certified-operators-catalog
+  annotations:
+    image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"catalogs:certified-operators"},"fieldPath":"spec.template.spec.containers[?(@.name==\"registry\")].image"}]'
```

Comment:
We don't pick up annotations from this manifest, only spec:
https://github.com/openshift/hypershift/blob/a50f62208e677dacb234f8c46c3ea64abb0dc4bb/control-plane-operator/controllers/hostedcontrolplane/olm/catalogs.go#L89
Either change the reconcile to also pick up annotations or set this in the reconcile.

Base on [this](https://docs.openshift.com/container-platform/4.11/openshift_images/using-imagestreams-with-kube-resources.html) it looks like you may also need to add

```
alpha.image.policy.openshift.io/resolve-names: '*'
```